config/unix-g++.cmake

```diff
@@ -34,9 +39,11 @@ check_c_compiler_flag(   "-march=native" HAS_MARCH_NATIVE )
 if( DEFINED CMAKE_CXX_COMPILER_ID )
   check_cxx_compiler_flag( "-Wnoexcept"    HAS_WNOEXCEPT )
   check_cxx_compiler_flag( "-Wsuggest-attribute=const" HAS_WSUGGEST_ATTRIBUTE )
-  check_cxx_compiler_flag( "-Wunused-local-typedefs"   HAS_WUNUSED_LOCAL_TYPEDEFS )
+  check_cxx_compiler_flag( "-Wunused-local-typedefs"   
+    HAS_WUNUSED_LOCAL_TYPEDEFS )
   #check_cxx_compiler_flag( "-Wunused-macros"           HAS_WUNUSED_MACROS )
-  check_cxx_compiler_flag( "-Wzero-as-null-pointer-constant" HAS_WZER0_AS_NULL_POINTER_CONSTANT )
+  check_cxx_compiler_flag( "-Wzero-as-null-pointer-constant" 
+    HAS_WZER0_AS_NULL_POINTER_CONSTANT )
```

Comment:
Is this really spelled ZER'0' with the number 0 not ZERO with the letter o? that seems strange.

Reply:
I'll fix this.  Actually, I will fix the spelling and comment out the check as it is currently unused.